fv3core/stencils/d_sw.py

```diff
@@ -744,10 +768,20 @@ def d_sw(
         )
         delnflux.compute_no_sg(wk, ut, vt, column_namelist["nord_v"], damp4, vort)
 
-    if column_namelist["d_con"] > dcon_threshold or spec.namelist.do_skeb:
-        damp = 0.25 * column_namelist["d_con"]
+    if kinetic_energy_fraction_to_damp > dcon_threshold or spec.namelist.do_skeb:
         heat_from_damping(
-            ub, vb, ut, vt, u, v, delp, fx, fy, gx, gy, heat_s, diss_e, damp
+            ub,
+            vb,
+            ut,
+            vt,
+            u,
+            v,
+            delp,
+            fx,
+            fy,
+            heat_s,
+            diss_e,
+            kinetic_energy_fraction_to_damp,
```

Comment:
This name is a bit long and turns one line into 12, but it is indeed descriptive